react_ws_src/src/views/ttt/GameMain.js

```diff
@@ -174,7 +173,7 @@ export default class SetName extends Component {
 
 		cell_vals[cell_id] = 'x'
 
-		TweenMax.from(this.refs[cell_id], 0.7, {opacity: 0, scaleX:0, scaleY:0, ease: Power4.easeOut})
+		this.animate_cell(cell_id);
```

Comment:
I noticed `TweenMax.from(this.refs[cell_id], 0.7, {opacity: 0, scaleX:0, scaleY:0, ease: Power4.easeOut})` was copy/pasted a bunch here, so I thought to be a bit more DRY, I'd move it to a function. It's probably something that could be abstracted even more into some kind of `animation_helper.js`